plugins/CoreVue/types/index.d.ts

```diff
@@ -257,6 +258,7 @@ declare global {
     Piwik_Transitions: TransitionsGlobal;
     SegmentedVisitorLog: SegmentedVisitorLogService;
     DataTable_RowActions_Registry: DataTableRowActionsRegisteryService;
+    Cloud
```

Comment:
doesn't this lack a type definition?

Reply:
:+1: I've now set this to `any`, it'll still need testing with the cloud plugin to make sure having a property definition doesn't interfere with loading the umd.